scripts/setupLocalIdler.sh

```diff
@@ -150,6 +156,43 @@ forwardToggleService() {
     echo "Toggle port forward stopped." >&2
 }
 
+###############################################################################
+# Retrieves the required OpenShift and Auth token from
+# Globals:
+#   JC_OPENSHIFT_API_TOKEN - OpenShift service account token used to monitor
+#                            OpenShift events
+#   JC_AUTH_TOKEN          - Auth token for the auth service
+# Arguments:
+#   None
+# Returns:
+#   None
+###############################################################################
+setTokens() {
+    pod=$(loc get pods -l deploymentconfig=jenkins-idler -o json | jq -r '.items[0].metadata.name')
+    if [ "${pod}" == "null" ] ; then
+        echo "WARN: Unable to determine Idler pod name"
+        return
+    fi
+
+    export JC_AUTH_TOKEN=$(loc exec ${pod} env | grep JC_AUTH_TOKEN | sed -e 's/JC_AUTH_TOKEN=//')
+    export JC_OPENSHIFT_API_TOKEN=$(loc exec ${pod} env | grep JC_OPENSHIFT_API_TOKEN | sed -e 's/JC_OPENSHIFT_API_TOKEN=//')
+}
+
+###############################################################################
+# Ensures login to OpenShift
+# Globals:
+#   None
+# Arguments:
+#   None
+# Returns:
+#   None
+###############################################################################
+login() {
+    [ -z "${DSAAS_PREVIEW_TOKEN}" ] && echo "DSAAS_PREVIEW_TOKEN needs to be exported." && printHelp && exit 1
+
+    loc login https://api.rh-idev.openshift.com -n dsaas-preview --token=${DSAAS_PREVIEW_TOKEN} >/dev/null
```

Comment:
@hferentschik Mind adding a check for correct namespace here after login? Might also make sense to exit on failure of login